uniffi_bindgen/src/bindings/python/templates/NamespaceLibraryTemplate.py

```diff
@@ -11,7 +11,13 @@ def loadIndirect():
     elif sys.platform == "darwin":
         libname = "lib{}.dylib"
     elif sys.platform.startswith("win"):
-        libname = "lib_{}.dll"
+        # As of python3.8, Windows does not search $PATH when looking for DLLs.
+        # It doesn't seem right to mess around with `os.add_dll_directory` and affect
+        # loading for the entire application, so for now, let's use a full path.
+        libname = os.path.join(
```

Comment:
mildly surprised that 3.8 changed how ctypes loads DLLs (I do know it changed how dynamic extension modules are loaded, but IIUC this isn't one of them) - and even then, the change was how *dependent* DLLs were located. IOW, I'm not sure this comment is 100% true (but also no evidence that it's not :) But regardless, loading a generic DLL via the full path when known is the right thing to do, so 👍